src/test/java/com/pngencoder/PngEncoderInputTypesTest.java

```diff
@@ -49,6 +55,8 @@ void showcaseMismatchInNumberOfChannels() {
 
         assertEquals(channels, 1);
         assertEquals(metaInfo.channels, 3);
+        // By default, we will fall back to encode the image as sRGB image
+        assertEquals(metaInfo.colorSpaceType, PngEncoderScanlineUtil.EncodingMetaInfo.ColorSpaceType.Rgb);
```

Comment:
Add more tests! `getRealGifImage()` is a real indexed-gif, which should succeed when converting directly!